src/reference/config/telemetry/exporter/exporter-otlp.md

```diff
@@ -0,0 +1,95 @@
+---
+shortTitle: otlp
```

Comment:
```suggestion
shortTitle: open telemetry
```

This is what shows in the sidebar so we can use something a little more descriptive. `open telemetry` should work.

Reply:
If you think `otlp` is used enough to be recognized by most people we can keep it like this.